exercise/submission_models.py

```diff
@@ -120,12 +120,12 @@ class Submission(UrlMixin, models.Model):
     A submission to some course exercise from one or more submitters.
     """
     STATUS = Enum([
-        ('INITIALIZED', 'initialized', _("Initialized")),
-        ('WAITING', 'waiting', _("In grading")),
-        ('READY', 'ready', _("Ready")), # graded normally
-        ('ERROR', 'error', _("Error")),
-        ('REJECTED', 'rejected', _("Rejected")), # missing fields etc
-        ('UNOFFICIAL', 'unofficial', _("No effect on grading")),
+        ('INITIALIZED', 'initialized', _('INITIALIZED')),
```

Comment:
Should these enum values use a prefix?

Reply:
I was of two minds on that.  "Rejected" is used elsewhere as well, and I thought that some of the others might be used somewhere else (at least in the future), but at the moment they aren't. Looking back at this, at least "initialized" and "error" would be good to have the "STATUS" prefix up front.
"Rejected" can or can not (depending on whether we want to have separate keys for both "rejected" uses, (or if we would want the points badge to use "STATUS_REJECTED" as well, which is also an option). What do you think?

"in grading" and "No effect on grading" are also difficult. They could be left without the prefix (as they are now, which would accent that it's not just the same word as the enum value), or the keys could just be "STATUS_WAITING" and "STATUS_UNOFFICIAL" while the msgstr would remain "In grading" and "No effect on grading". Personally I'd probably prefer the latter. What do you think?